src/puppetlabs/trapperkeeper/services/webserver/jetty10_config.clj

```diff
@@ -83,6 +83,7 @@
 (def default-protocols ["TLSv1.3" "TLSv1.2"])
 (def default-client-auth :need)
 (def default-allow-renegotiation false)
+(def sni-required false)
```

Comment:
unused?

Reply:
used in the test

Reply:
ah don't you want to use it instead of false on https://github.com/puppetlabs/trapperkeeper-webserver-jetty10/pull/22/files#diff-4083ca6599d7f9eb86bdfe81253ffde52f708b4e90cb2f7452e8deef6cbb57b2R425 ? If not why not just define in the test?

Reply:
yeah, good catch, change incoming...